mobile/library/proguard.txt

```diff
@@ -2,10 +2,12 @@
 -dontwarn kotlin.**
 -dontwarn org.jetbrains.annotations.NotNull
 -dontwarn org.jetbrains.annotations.Nullable
+-dontwarn com.google.protobuf.**
```

Comment:
just curious, why were these changes needed?

Reply:
That caused the proguard build to fail.